src/main/kotlin/org/jetbrains/research/testspark/collectors/TestGenerationStartedCollector.kt

```diff
@@ -0,0 +1,31 @@
+package org.jetbrains.research.testspark.collectors
+
+import com.intellij.internal.statistic.eventLog.EventLogGroup
+import com.intellij.internal.statistic.eventLog.events.EnumEventField
+import com.intellij.internal.statistic.eventLog.events.EventFields
+import com.intellij.internal.statistic.service.fus.collectors.CounterUsagesCollector
+import org.jetbrains.research.testspark.data.CodeType
+import org.jetbrains.research.testspark.data.Technique
+
+/**
+ * This class represents a collector for tracking the event of test generation being started.
+ *
+ * @property technique The EnumEventField<Technique> instance representing the technique used for test generation.
+ * @property level The EnumEventField<CodeType> instance representing the code type for which test generation is performed.
+ */
+class TestGenerationStartedCollector : CounterUsagesCollector() {
+    private val groupId = "tests.set"
+    private val group = EventLogGroup(groupId, 1)
+
```

Comment:
What does a constant `1` stand for here? It's better to have a named constant for it.